.github/workflows/test.yml

```diff
@@ -131,6 +131,25 @@ jobs:
         with:
           bundler-cache: true
 
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'yarn'
+
+      - name: Cache Node modules
+        id: node-modules-cache
+        uses: actions/cache@v3
+        with:
+            path: node_modules
+            key: ${{ runner.os }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+            restore-keys: |
+              ${{ runner.os }}-nodemodules-
+
+      - name: Yarn install
+        if: steps.node-modules-cache.outputs.cache-hit != 'true'
+        run: yarn
```

Comment:
I remember from v2, they already support dependencies caching. Could you remove these lines and retry?

Reply:
@hoangmirs Nice 👍